### ISSUE TITLE
AttributeError: 'project' has no attribute 'myclass'

### ISSUE THREAD

**Issue by mipcmhone6s** [amymcgovern/pyparrot#203]
![맘보 디렉트 실행할때 오류화면](https://user-images.githubusercontent.com/38872897/89115578-d9bde200-d4c4-11ea-82f7-4b0515ac280f.png)

Hello! @amymcgovern 
I really like this project.

I ran "demoMamboDirectFlight.py", but I met this issue.

Does anyone have solved it?

**Comment by OhSeungBeen** [comment]
I solved it myself.
I changed class -> myclass in xml.

Thanks.

**Comment by sergio14** [comment]
Hi, it seems like this is related to untangle updates. You will need to edit: at least line 163 in DroneSensorParser.py  and 45 in DroneCommandParser,  if you take a look  you will note that where my_file.project.myclass  need to be replace by my_file.project.get_elements() in order to align to new untangle parser object.
Best.

**Comment by toncho11** [comment]
> Hi, it seems like this is related to untangle updates. You will need to edit: at least line 163 in DroneSensorParser.py and 45 in DroneCommandParser, if you take a look you will note that where my_file.project.myclass need to be replace by my_file.project.get_elements() in order to align to new untangle parser object. Best.

Yes. This fixed the problem, but why is it not updated in the main code? Maybe because there is a suggestion for another XML parsing library by ShusenLiu?

**Comment by amymcgovern** [comment]
I haven't updated as I haven't been supporting the project anymore due to not having working parrot drones anymore (since parrot stopped supporting them)

**Comment by toncho11** [comment]
I see. You mean the Minidrones are no longer supported?

**Comment by amymcgovern** [comment]
Yes, parrot stopped supporting anything except the anafi
